extensions/dicom-sr/src/tools/DICOMSRDisplayTool.js

```diff
@@ -280,10 +280,9 @@ export default class DICOMSRDisplayTool extends BaseTool {
 }
 
 function _getTextBoxLinesFromLabels(labels) {
-  // TODO -> max 2 for now, need a generic solution for this!
-
-  const labelLength = Math.min(labels.length, 2);
+  // TODO -> max 3 for now (label + shortAxis + longAxis), need a generic solution for this!
```

Comment:
Do we have a follow-up issue for this? If so, can we add it in comments? If not, can we create it?

Reply:
I think its out of scope for the foresable future, but I can create the issue.